packages/erxes-ui/src/components/richTextEditor/extensions/index.ts

```diff
@@ -1,7 +1,9 @@
 export { Link } from './Link';
 export { FontSize } from './FontSize';
 export { ImageResize } from './Image';
-export { MentionExtended } from './Mention';
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **🚨 issue (security):** Ensure exported names are consistent with the module's exported name.